likecoin/admin/views/metabox.php

```diff
@@ -139,19 +139,37 @@ function likecoin_add_publish_meta_box( $publish_params ) {
 					<?php } ?>
 				</td>
 			</tr>
+			<?php if ( 'Published (testnet)' === $iscn_status['status'] ) { ?>
+				<tr id="likecoin_submit_iscn_testnet">
+					<th><label><?php esc_html_e( 'ISCN (Testnet) Status', LC_PLUGIN_SLUG ); ?></label></th>
+					<td id="lcISCNStatus">
+						<span id="lcISCNPublish" style="display:<?php echo esc_attr( empty( $status['ipfs']['url'] ) ? 'none' : '' ); ?>">
+							<a rel="noopener" target="_blank" href="<?php echo esc_url( $iscn_status['url'] ); ?>">
+								<?php echo 'Published'; ?>
+							</a>
+						</span>
+					</td>
+				</tr>
+			<?php } ?>
 			<tr id="likecoin_submit_iscn">
-				<th><label><?php esc_html_e( 'ISCN (Testnet) Status', LC_PLUGIN_SLUG ); ?></label></th>
+				<th><label><?php esc_html_e( 'ISCN (Main) Status', LC_PLUGIN_SLUG ); ?></label></th>
 				<td id="lcISCNStatus">
-					<?php if ( ! empty( $iscn_hash ) ) { ?>
+					<?php if ( ! empty( $iscn_hash ) && 'Published' === $iscn_status['status'] ) { ?>
 						<a rel="noopener" target="_blank" href="<?php echo esc_url( $iscn_status['url'] ); ?>">
 							<?php echo esc_html( $iscn_status['status'] ); ?>
 						</a>
-					<?php } else { ?>
+					<?php } elseif ( 'Published (testnet)' === $iscn_status['status'] ) { ?>
```

Comment:
Similarly, should not use hard code string to compare, should use a flag or dedicated key for testnet